.github/workflows/build.yml

```
@@ -0,0 +1,82 @@
+name: Build
+
+on:
+  push:
+    ignore-branches: [main, future]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  # The Pulumi version doesn't matter much, it's just used for sanity testing
+  # but worth to update every once in a while
+  PULUMI_VERSION: "v3.46.0"
+  REPOSITORY_BASE_PATH: spaceliftbuild
+
+jobs:
+  build:
+    if: ${{ github.ref_type != 'tag' }}
+    name: 🚧 ${{ matrix.lang }} on ${{ matrix.arch}} 
```

tiny tiny tiny tiny nitpick: this spacing is off around these curly braces.
```suggestion
    name: 🚧 ${{ matrix.lang }} on ${{ matrix.arch }} 
```